tests/test_codesyncd/buffer_handler.test.js

```diff
@@ -1,8 +1,8 @@
-import fs from "fs";
+import fs, { read } from "fs";
```

Comment:
What is this import doing?

Reply:
`read`  must have been imported  automatically, removing